source/agora/network/RPC.d

```diff
@@ -131,7 +131,7 @@ public class RPCClient (API) : API
         public Duration retry_delay  = 1.seconds;
 
         ///
-        public uint concurrency = 3;
+        public uint concurrency = 1;
```

Comment:
Why this change ?

Reply:
We use `Config.init` for incoming connections, for which we can't enable concurrency since we don't know the server address. If we do, client tries to iniate a new connection to an unindentified server and fails as expected.

https://github.com/bosagora/agora/commit/c5e36c49dea4a569f398974e9974d015e7116b31#diff-f5162195974e34bb5a3926eac58610027532467ff12d53569827d5867072b62cR199

Reply:
We already set it to 3 for outgoing RPC connections.